tests/core/pyspec/eth2spec/gen_helpers/gen_base/gen_runner.py

```diff
@@ -216,6 +227,28 @@ def output_part(out_kind: str, name: str, fn: Callable[[Path, ], None]):
         if span > TIME_THRESHOLD_TO_PRINT:
             summary_message += f" in {span} seconds"
         print(summary_message)
+    diagnostics = {
+        "collected_test_count": collected_test_count,
+        "generated_test_count": generated_test_count,
+        "skipped_test_count": skipped_test_count,
+        "test_identifiers": test_identifiers,
+        "durations": [f"{span} seconds"],
```

Comment:
could you print the `generator_name` of each span so we can see which is which?

Reply:
yeah, I think an obvious next step is to improve the fidelity of the diagnostics

do you think doing this would block merging this PR?

Reply:
no, it's fine to add it later.